internal/api/v1/router.go

```diff
@@ -122,10 +123,24 @@ var Routes = routes.NamedRoutes{
 	"ServiceReplace": put("/namespaces/:namespace/services/:service", errorHandler(service.Controller{}.Replace)),
 }
 
+var WsRoutes = routes.NamedRoutes{
+	"AppExec":     get("/namespaces/:namespace/applications/:app/exec", errorHandler(application.Controller{}.Exec)),
+	"AppLogs":     get("/namespaces/:namespace/applications/:app/logs", application.Controller{}.Logs),
+	"StagingLogs": get("/namespaces/:namespace/staging/:stage_id/logs", application.Controller{}.Logs),
+}
+
 // Lemon extends the specified router with the methods and urls
 // handling the API endpoints
 func Lemon(router *gin.RouterGroup) {
 	for _, r := range Routes {
 		router.Handle(r.Method, r.Path, r.Handler)
 	}
 }
+
+// Spice extends the specified router with the methods and urls
+// handling the API endpoints
```

Comment:
```suggestion
// handling the Websocket API endpoints
```

And :rofl: for keeping in theme, somewhat.